e2e_tests/docker_image/parsec-service-test-all.Dockerfile

```diff
@@ -161,3 +161,18 @@ ENV SPIFFE_ENDPOINT_SOCKET="unix:///tmp/agent.sock"
 
 # Add safe.directory configuration to access repos freely
 RUN git config --global --add safe.directory '*'
+
+# Install latest Trusted Services libraries. The previously installed
+# libraries are old and necessary for ./generate-keys.sh which uses 
+# Parsec 1.0.0 version that is incompatible with newer libts APIs. 
+RUN rm /usr/local/lib/libts.so* /usr/local/lib/libprotobuf-nanopb.a /usr/local/lib/libmbedcrypto.a 
+RUN git clone https://git.trustedfirmware.org/TS/trusted-services.git --branch integration \
```

Comment:
In the latest Yocto TS 1.0 updates the branch was changed from integration to main:
https://git.yoctoproject.org/meta-arm/tree/meta-arm/recipes-security/trusted-services/trusted-services-src.inc

Reply:
You might also need to update [trusted-services-vendor] to the main branch

Reply:
Thanks for the comment Anton. Will update this.